libafl_cc/Cargo.toml

```diff
@@ -21,3 +21,5 @@ which = "4.2.2"
 glob = "0.3"
 
 [dependencies]
+libafl = { path = "../libafl", version = "0.7.1", default-features = false, features = [] }
```

Comment:
why do you need libafl as dep here? It's never used in your patches

Reply:
Removed!